Entity/User.php

```diff
@@ -0,0 +1,466 @@
+<?php
```

Comment:
declare strict

Reply:
migrated as-is from local-user, strict typing will give issue on the $authTokens variable which I won't readd in this pr